### PR TITLE
Add actix-http support for actix error messages

### PR DESCRIPTION
**Pull request by dunnock**
As part of actix:0.10 release actix will not have "http" feature flag https://github.com/actix/actix/issues/321 which allowed to break circular dependency. Though actix does not support handling of actor's error messages via ResponseError anymore. 

This PR introduces "actors" feature flags which would re-add support for handling of MailboxError and ResolverError (with "resolver" flag).

### CODE REVIEW

File: actix-http/Cargo.toml [hunk 1]
```diff
@@ -15,7 +15,7 @@ license = "MIT/Apache-2.0"
 edition = "2018"
 
 [package.metadata.docs.rs]
-features = ["openssl", "rustls", "failure", "compress", "secure-cookies"]
+features = ["openssl", "rustls", "failure", "compress", "secure-cookies","actix"]
```
Comment (robjtede) [inline actix-http/Cargo.toml]: feature name is actors or actix ?

**Review by robjtede — CHANGES_REQUESTED** [review]

File: actix-http/Cargo.toml (hunk 1 above)
Comment (JohnTitor) [reply actix-http/Cargo.toml]: Yeah, should be `actors`.

```suggestion
features = ["openssl", "rustls", "failure", "compress", "secure-cookies","actors"]
```

Comment (dunnock) [reply actix-http/Cargo.toml]: Oh, right, sorry, it was late night fix. I have pushed with few more documentation changes.

**Review by robjtede — APPROVED** [review]

**Review by JohnTitor — APPROVED** [review]

**Comment by JohnTitor** [comment]
Thanks!

**Pull request merged**